google/cloud/storage/internal/storage_stub_factory_test.cc

```diff
@@ -55,7 +55,7 @@ class StorageStubFactory : public ::testing::Test {
                         google::protobuf::Message const& request) {
     return validate_metadata_fixture_.IsContextMDValid(
         context, method, request,
-        google::cloud::internal::ApiClientHeader("generator"));
+        google::cloud::internal::HandCraftedLibClientHeader());
```

Comment:
Should this be calling `storage::x_goog_api_client()`?

Reply:
Done.